openslides/motions/models.py

```diff
@@ -339,7 +339,11 @@ def get_new_version(self, **kwargs):
         attributes, it is populated with the data from the last version
         object if such object exists.
         """
-        new_version = MotionVersion(motion=self, **kwargs)
+        if self.pk is None:
+            # Do not reference the MotionVersion object to an unsaved motion
```

Comment:
This is save. See the line below the deleted comment in my last commit.

Before django 1.8 it looks like a bug, because it was possible to set a unsaved motion for the new MotionVersion, but it was not used. Now, it is not possible to set the unsaved motions. So it is clear, that the line is needed.